packages/web3-providers-http/types/tslint.json

```diff
@@ -5,7 +5,6 @@
         "no-import-default-of-export-equals": false,
         "file-name-casing": [true, "kebab-case"],
         "whitespace": false,
-        "no-unnecessary-class": false,
-        "no-redundant-jsdoc": false
+        "no-unnecessary-class": false
```

Comment:
Did you never have this strange error message? dtslint was not always happy with having the same copyright/author comment @joshstevens19 knows this error too.

Reply:
I've resolved this because it does only happen sometimes and I will investigate further the next time it happens.